css/demo.css

```diff
@@ -0,0 +1,304 @@
+        color: transparent;
+        text-shadow: none;
+      }
+    }
+  }
+}
+
+[data-popover-title] {
+  font-size: var(--title-size);
+
+  @container main-header (width >=350px) {
+    --title-size: var(--step-8);
+  }
+
+  @container main-header (width >=1200px) {
+    --title-size: var(--step-9);
+  }
+
+  @container main-header (width >=1650px) {
+    --title-size: var(--step-10);
+  }
+}
+
+[data-popover-title~='attribute'] {
+  font-weight: 400;
+}
+
+.subtitle {
+  font-family: var(--serif);
+  font-style: italic;
+  font-weight: 400;
+}
+
+.description {
+  max-inline-size: 65ch;
+
+  @container main-header (width >=830px) {
+    min-inline-size: 35ch;
+  }
+}
+
+.demo-section {
+  display: grid;
+  gap: var(--gap);
+  grid-template-columns: minmax(auto, 1fr) minmax(0, 57ch) minmax(auto, 1fr);
+  justify-items: center;
+  margin-block-start: var(--double-gap);
+  position: relative;
+
+  &::before {
+    border-block-end: thin solid var(--border-color);
+    content: '';
+    inset: 0;
+    inset-block-end: auto;
+    position: absolute;
+  }
+
+  &::after {
+    border-block-start: thin solid var(--border-color);
+    content: '';
+    position: absolute;
+    inset: 0 0 calc(var(--gap) * -2);
+    inset-block-start: auto;
+  }
+}
+
+.demo-item {
+  display: grid;
+  grid-column: 2;
+  grid-template: 'title' min-content 'note' auto 'popover-button' min-content 'code' 1fr / minmax(
+      20ch,
+      1fr
+    );
+  inline-size: 100%;
+}
+
+[data-header] {
+  grid-area: title;
+  position: relative;
+
+  [aria-hidden]:any-link {
+    display: inline-block;
+    filter: grayscale(var(--grayscale, 100%));
+    left: -1.25em;
+    position: absolute;
+    text-decoration: none;
+    transform: scale(var(--scale, 0.75));
+    transition: all 200ms ease-in-out;
+  }
+
+  [aria-hidden]:hover,
+  [aria-hidden]:focus,
+  [aria-hidden]:active {
+    --grayscale: 0;
+    --scale: 1;
+    --outline-offset: 0.025em;
+  }
+}
+
+.button-group {
+  display: grid;
+  gap: var(--gap);
+}
+
+.note {
+  grid-area: note;
+  margin-block-end: var(--double-gap);
+}
+
+[data-popover] {
+  --popover-offset: 150px;
+}
+
+[data-popover]:popover-open {
+  align-items: center;
+  border: 1px solid var(--border-color);
+  box-shadow: 0 3px 3px var(--shadow);
+  color: var(--popover-color, var(--text));
+  display: flex;
+  font-family: var(--sans-serif);
+  flex-wrap: wrap;
+  gap: var(--shim);
+
+  @media (width <=780px) {
+    inline-size: auto;
+    justify-content: center;
+    margin-inline: var(--shim);
+  }
+}
+
+/* for browsers that don't support popovers */
+[data-popover].\:popover-open {
+  align-items: center;
+  border: 1px solid var(--border-color);
+  box-shadow: 0 3px 3px var(--shadow);
+  color: var(--popover-color, var(--text));
+  display: flex;
+  font-family: var(--sans-serif);
+  flex-wrap: wrap;
+  gap: var(--shim);
+
+  @media (width <=780px) {
+    inline-size: auto;
+    justify-content: center;
+    margin-inline: var(--shim);
+  }
+}
```

Comment:
Could this be combined with the above duplicate set?

Reply:
I thought it could, but saw this in this `Readme`:

> This is important because if you intend to style a popover using
>     `.\:popover-open` it will need to be a separate declaration. For example,
>     `[popover]:popover-open, [popover].\:popover-open` will not work.